sidekick_core/lib/src/template/sidekick_package.template.dart

```diff
@@ -0,0 +1,269 @@
+import 'package:recase/recase.dart';
+
+import 'package:sidekick_core/sidekick_core.dart';
+import 'package:sidekick_core/src/template/download_dart.sh.template.dart';
+import 'package:sidekick_core/src/template/entrypoint.template.dart';
+import 'package:sidekick_core/src/template/install.sh.template.dart';
+import 'package:sidekick_core/src/template/run.sh.template.dart';
+import 'package:sidekick_core/src/template/sidekick_config.sh.template.dart';
+
+class SidekickTemplate {
+  void generate(SidekickTemplateProperties props) {
```

Comment:
In my opinion we can make all the shell scripts executable here instead of in `sidekick/InitCommand`- but either way is fine for me :)